yarn.lock

```diff
@@ -15994,6 +15999,13 @@ vue-template-es2015-compiler@^1.6.0, vue-template-es2015-compiler@^1.9.0:
   resolved "https://registry.yarnpkg.com/vue-template-es2015-compiler/-/vue-template-es2015-compiler-1.9.1.tgz#1ee3bc9a16ecbf5118be334bb15f9c46f82f5825"
   integrity sha512-4gDntzrifFnCEvyoO8PqyJDmguXgVPxKiIxrBKjIowvL9l+N66196+72XVYR8BBf1Uv1Fgt3bGevJ+sEmxfZzw==
 
+vue2-datepicker@^3.11.1:
```

Comment:
This looks pretty abandoned.... plus it will break with vue3. Are there any javascript-only implementations?

Reply:
I can look for a pure JS version, was hoping to use a vue specific one but can look into another one. Had a couple people recommend this one to me.

Reply:
Have you had a look at Vuetify's date pickers?
Might be nice, and you should be able to only use the date pickers, so it wont mess up the bundle size.